contracts/Administratable.sol

```diff
@@ -33,18 +36,14 @@ contract Administratable {
    * @notice onlyAdminOrRole checks that the caller is either the Admin or the provided role.
    * @param adminContractAddress supplied EndaomentAdmin address
    * @param role The role to require unless the caller is the owner. Permitted
-   * roles are admin (0), accountant (2), and reviewer (3).
+   * roles are ADMIN (6), ACCOUNTANT (2), REVIEWER (3), FUND_FACTORY (4) and ORG_FACTORY(5).
    */
   modifier onlyAdminOrRole(address adminContractAddress, IEndaomentAdmin.Role role) {
     _onlyAdminOrRole(adminContractAddress, role);
     _;
   }
-  
-  function _onlyAdminOrRole(address adminContractAddress, IEndaomentAdmin.Role role) 
-    private
-    view
-  {
 
+  function _onlyAdminOrRole(address adminContractAddress, IEndaomentAdmin.Role role) private view {
```

Comment:
This function seems to be the only one without docstrings, otherwise this PR looks good!

Reply:
The docstring would be the same as the modifier above. Robbie do you want to add a brief comment there?